SU-HSDO/suhumsci
# Link plugin in CKEditor should not add redundant title attribute

# READY FOR REVIEW

## Summary
Updates Linkit configuration so if title is not set then the `title` attribute is ignored 

## Urgency
low

## Steps to Test
1. Create a basic page
2. Use the link plugin in CKEditor to create a link to an internal node
3. Do not input a title
4. Use the link plugin to create a link to a separate internal node
5. Input a title
6. Save the content
7. Link in saved content from steps 2-3 does not have the `title` attribute (check by hovering or inspecting html)
8. Link in saved content from steps 4-5 does have the `title` attribute set correctly (check by hovering or inspecting html)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
